### ISSUE TITLE
Idea: Notify pledges to recipients

### ISSUE THREAD

**Issue by jelv** [liberapay/liberapay.com#598]
Notify pledges to recipients on there own social network account. Could be from an official Liberapay account or the person who pledged.

**Comment by Changaco** [comment]
This is a delicate matter because overdoing it can be perceived as spam. I think that for now we could simply inform donors that we don't notify pledgees, that they can do it themselves if they want, but that they shouldn't overdo it either.

**Comment by jelv** [comment]
I agree that automatic notification is spam and should be avoided. But a button for the Liberapay pledger so he can easly notify the recipient on the right social network (profile link or compose message screen) seems like a good solutions. How else is the recipient going to know about the pledge?

**Comment by mattbk** [comment]
Could build an opt-out mechanism that works only with the pledge system. Or keep track of accounts that have been automatically notified and use that to prevent additional notifications from the pledge system after the first time.

**Comment by rugk** [comment]
I would not say an automatic information would be bad. Just do it rarely, e.g. when the first user pledges something, when the pledge surpassed a certain amount or so.

Just use the channel you know: E.g. when a user pledges to a Twitter account, send them a tweet.

Also you could (with the user's consent) do it in the user's name. Or, as said, offer a simple (tweet) button…
